src/Builder/ClassGenerator/ClassGenerator.php

```diff
@@ -186,19 +186,21 @@ protected function resolveModuleDir(string $organisation, string $module): strin
      */
     protected function findLicenseBlock(string $moduleDir): string
     {
-        if (file_exists($moduleDir . '.license')) {
-            return file_get_contents($moduleDir . '.license');
+        if (!file_exists($moduleDir . '.license')) {
+            return '';
```

Comment:
find usually is nullable in return

Reply:
I would have done it the other way around, as empty string means null here, you are faking it only

Reply:
return null, and check in user code if it is null or not. thats cleaner.